Week2/3.js

```diff
@@ -0,0 +1,11 @@
+let x;
+
+console.log('the value of variable x will be a number');
```

Comment:
are you sure that the value is a number? I don't think it will be

Reply:
I misunderstood it, you're right. it should have been 'the value of variable x will be an integer'.

Reply:
Ah, close! you can `console.log` it for yourself and see the value:)